pinot-controller/src/main/java/com/linkedin/pinot/controller/ControllerConf.java

```diff
@@ -113,6 +118,10 @@ public String getHelixClusterName() {
     return (String) getProperty(HELIX_CLUSTER_NAME);
   }
 
+  public String getControllerVipPort() {
```

Comment:
Can we make the controller.vip.port config optional and default it to controller port?